src/PolyhedralGeometry/PolyhedralComplex/properties.jl

```diff
@@ -364,6 +364,9 @@ _vertex_and_ray_indices(::Val{_maximal_polyhedron}, PC::PolyhedralComplex) =
 
 Return the maximal polyhedra of `PC`
 
+Optionally `IncidenceMatrix` can be passed as a first argument to receive the
+incidence matrix specifying the maximal polyhedra of `PC`.
```

Comment:
```suggestion
Optionally `IncidenceMatrix` can be passed as a first argument to return the
incidence matrix specifying the maximal polyhedra of `PC`.
```
I would prefer `return` instead of `receive`.
This should probably refer to `vertices_and_rays` (which is currently undocumented) to explain the column indices.